### ISSUE TITLE
Optimize Views for Hozontal mode or should we disable it?

### ISSUE THREAD

**Issue by AndiAndreas** [mobileappdevhm/dev-team-1-cie-app-in-flutter#47]

**Comment by EduardBartolovic** [comment]
could be handy on some screens

**Comment by AndiAndreas** [comment]
Which one do you mean? Timetable is the only one I think.

**Comment by failita** [comment]
Settings too. 

I don't think it's optimal to use our current version in horizontal mode. 

We could have to have a different design for each page. Maybe use gridView instead of listView for horizontal mode and redesign other pages that are not using listView. 

Or maybe we could just disable it. I don't think anybody is going to use this app in horizontal mode anyway.

**Comment by heatsink** [comment]
I'm personally a fan of disabling the Horizontal mode assuming most people will be using the app on a phone and not devices like tablets.

However, it wouldn't be too much work to extend functionality to optimize for a horizontal view if we decide that this is necessary. If we do decide to optimize for it, I think it would be confusing to enable it only on certain pages so I think we would have to prepare it for all pages.

**Comment by RedTo** [comment]
Yeah, I agree. If we allow horizontal mode we should build it for all pages. Let's discuss this topic in the next sprint.

**Comment by AndiAndreas** [comment]
Status? We only implement horizontal as discussed right?

**Comment by RedTo** [comment]
I think the easiest way is to disable it for now and maybe add it later, after the sprint if our app is the one Ms Socher would like to have in the app stores.

**Comment by heatsink** [comment]
I agree, we should disable horizontal view for now.
